canonical/juju-dashboard
# WD-8406 - Add secrets in the config panel

## Done

- Made secrets form reusable.
- Add the ability to add secrets in the config panel.

## QA

- Go to an application and open the config panel.
- Click the secrets menu icon and you should see an "Add a secret" button.
- Click the button and it should open the add-secret form.
- Fill out the form and submit.
- The form should close and the secret URI should get entered into the config field.

## Details

- https://warthogs.atlassian.net/browse/WD-8406
